### PR TITLE
Update to API 7.2.0

### PR DESCRIPTION
**Pull request by Inscrutable**
Fix broken link to Mixin examples. New link from SpongeAPI 7.1.6

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/99c37ba/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/99c37ba/contributing/implementation/mixins.html
- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/99c37ba/plugin/project/gradle.html
- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/99c37ba/plugin/project/index.html
- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/99c37ba/plugin/project/maven.html
- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/99c37ba/server/getting-started/configuration/sponge-conf.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/829#issuecomment-612406337), not to the direct url to the preview._

**Comment by Inscrutable** [comment]
I plan to update Docs to SpongeAPI 7.2. The following things may also need updating:
- plugin/project/gradle
- plugin/project/index
- plugin/project/maven
- server/getting-started/configuration/sponge-conf
- plugin/debugging
- plugin/commands/arguments
- extend copyright to 2020

**Comment by Inscrutable** [comment]
The only step remaining from the list above is to update the sponge.conf page to API 7.2.0 (preferably the recommended build). Didn't we just update that page? ;)

**Comment by ST-DDT** [comment]
Since I recently touched that page, I also updated the embedded global conf using:
 `spongeforge-1.12.2-2838-7.2.0.jar`

File: source/contributing/implementation/mixins.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ is available at the `Mixin Wiki <https://github.com/SpongePowered/Mixin/wiki/>`_
 mixins themselves are actually discussed.*
 
 If you're looking to get started writing mixins, we also strongly recommend carefully examining all of the examples in
-the `SpongeCommon repository <https://github.com/SpongePowered/SpongeCommon/tree/stable-7/src/example/java/org/spongepowered>`__ which
+the `SpongeCommon repository <https://github.com/SpongePowered/SpongeCommon/tree/v7.1.6/src/example>`__ which
```
Comment (ST-DDT) [inline source/contributing/implementation/mixins.rst:20]: What happened to the examples? 👀 Gone without replacement?

**Review by ST-DDT — APPROVED** [review]

**Comment by ST-DDT** [comment]
@Inscrutable If you wish to keep your PR "clean" I could move the javadoc fixes to another PR.

File: source/contributing/implementation/mixins.rst (hunk 1 above)
Comment (Inscrutable) [reply source/contributing/implementation/mixins.rst:20]: @dualspiral gave me that link. I take no responsibility for the content ;)

**Comment by Inscrutable** [comment]
> @Inscrutable If you wish to keep your PR "clean" I could move the javadoc fixes to another PR.

I'm happy for you to bundle it all in here if it's relevant to the 7.2.0 update. I'm sure there will be more anyway, many things are out of date.

**Pull request merged**